docs/_docs/user-interface/statistics.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Column's Statistics
+permalink: /docs/statistics/
```

Comment:
A corresponding entry to the docs.yml file is missing if you want this nice page to be accessible by the public :)

Reply:
@rhuille an entry in the docs.yml file is still missing (your doc will not be indexed without that)